src/aux-common/Formulas/formula-lib.ts

```diff
@@ -934,7 +934,7 @@ function remote(event: BotAction, selector?: SessionSelector) {
  *
  * @example
  * // Send the player to the "welcome" context.
- * player.goToContext("welcome");
+ * player.goToContext("https://freesound.org/data/previews/58/58277_634166-lq.mp3");
```

Comment:
@shaneYeti This looks like it should go in the example for the `playSound()` function